AsyncDisplayKit/ASEditableTextNode.h

```diff
@@ -11,7 +11,7 @@
 
 @protocol ASEditableTextNodeDelegate;
 
-/// @abstract ASEditableTextNode implements a node that supports text editing.
+/// @abstract ASEditableTextNode implements a view-backed node that supports text editing.
```

Comment:
Optional, but might be better to say as a separate sentence "Does not support layer-backing".